app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -104,7 +104,7 @@ $include-html-global-classes: $include-html-classes;
 // $smoke       : #EEEEEE;
 // $gainsboro   : #DDDDDD;
 // $iron        : #CCCCCC;
-// $base        : #AAAAAA;
+$base        : #AAAAAA;
```

Comment:
Color `#AAAAAA` should be written as `#aaaaaa`<br>Variable names should be followed immediately by a colon